File: Anna Permyakova/code.js
class Greeter {
    greet() {
        const greetings = ["Hello", "Hola", "Bonjour", "Привет", "こんにちは"];
        greetings.forEach(greeting => console.log(greeting));
    }
}

const greeter = new Greeter();
greeter.greet();

File: Anna Permyakova/code.cpp
#include <iostream>

class Greeter {
public:
    void Greet() {
        std::string greetings[] = {"Hello", "Hola", "Bonjour", "Привет", "こんにちは"};
        for (const auto& greeting : greetings) {
            std::cout << greeting << std::endl;
        }
    }
};

int main() {
    Greeter greeter;
    greeter.Greet();
    return 0;
}
